### ISSUE TITLE
`wrapLines=true` removes leading whitespace before `<`

### ISSUE THREAD

**Issue by dcsaszar** [conorhastings/react-syntax-highlighter#56]
### Input
```jsx
import SyntaxHighlighter from 'react-syntax-highlighter';
<SyntaxHighlighter language="javascript" wrapLines={true}>
  { " (\n  <br/>\n );" }
</SyntaxHighlighter>
```

### Actual output
```
 (
<br/>
 );
```
### Expected output
```
 (
  <br/>
 );
```

The output is as expected with `wrapLines={false}`.
Update: It's also correct with `language="html"`.

**Comment by conorhastings** [comment]
@dcsaszar thanks, will try to take a look this weekend

**Comment by conorhastings** [comment]
@dcsaszar fixed in `v5.1.3`
